samples/XAMLCatalog/XAMLCatalog/UIKitControls/UILabelViewController.h

```diff
@@ -17,5 +17,30 @@
 
 #import "MenuTableViewController.h"
 
-@interface UILabelViewController : MenuTableViewController
+@interface UILabelViewController : UIViewController <UITextFieldDelegate>
+@property (nonatomic) UITextField* text;
+@property (nonatomic) UISlider* fontSize;
+@property (nonatomic) UITextField* x;
+@property (nonatomic) UITextField* y;
+@property (nonatomic) UITextField* w;
+@property (nonatomic) UITextField* h;
```

Comment:
If you used TestEnabled- controls, you could have leveraged the same shorthand used in UIButton inset tests. We could have built the a single control for all these types (CGRect, UIEdgeInsets,etc.)